model/resource/oci.yaml

```
@@ -0,0 +1,24 @@
+groups:
+  - id: oci.manifest
+    prefix: oci.manifest
+    type: resource
+    brief: >
+      An OCI image manifest.
+    attributes:
+      - id: digests
+        type: string[]
```

```suggestion
      - id: digest
        type: string
```


Image/artifact has one manifest (which can have multiple nested manifests or layers)